roles/network_plugin/kube-router/defaults/main.yml

```diff
@@ -0,0 +1,37 @@
+---
+# Enables Pod Networking -- Advertises and learns the routes to Pods via iBGP
+kube_router_run_router: true
+
+# Enables Network Policy -- sets up iptables to provide ingress firewall for pods
+kube_router_run_firewall: true
+
+# Enables Service Proxy -- sets up IPVS for Kubernetes Services
+# TODO(jjo): this setting currently requires also:
+# - no kubeadm_enabled
```

Comment:
If this is yet not supported, do a check in the pre-install stage to make sure it does not happen?

Reply:
Thanks for the suggestion:
- added assertion on `not kubeadm_enabled`
- added set_fact to override `kube_proxy_mode: none` if `kube-router` is set to replace it